Assets/Scripts/Player.cs

```diff
@@ -86,15 +86,21 @@ public class Player : MonoBehaviour {
     void Update() {
         //Get player position.  Different in VR
         if (vrActive) {
-            int[] hexCoords = HexConst.CoordToHexIndex(new Vector3(transform.position.x, uiController.transform.position.y, transform.position.z));
+			int[] hexCoords = HexConst.CoordToHexIndex(new Vector3(uiController.transform.position.x, uiController.transform.position.y - 1, uiController.transform.position.z));
```

Comment:
This will break with the new minimap changes.  I move the UIController to center the minimap on the hex.  Maybe instead grab the position from the Player wrapper object (the thing that wraps FPSController, bounding box, UI, etc).  I'm not gonna merge since I know it will break, lol